### PR TITLE
(MAINT) Fix install_puppet_from msi_on non-cygwin

### PR DESCRIPTION
**Pull request by petems**
Also adds spec for method too

### CODE REVIEW

File: lib/beaker/dsl/install_utils.rb [hunk 1]
```diff
@@ -900,15 +900,21 @@ def install_puppet_from_msi( host, opts )
         else
           dest = "C:\\Windows\\Temp\\#{host['dist']}.msi"
 
-          on host, "set PATH=\"%PATH%;#{host['puppetbindir']}\""
-          on host, "setx PATH \"%PATH%;#{host['puppetbindir']}\""
+          set_path_string = host['puppetbindir'].gsub(/"/,'')
+
+          on host, "set PATH=\"%PATH%;#{set_path_string}\""
+          on host, "setx PATH \"%PATH%;#{set_path_string}\""
```
Comment (anodelman) [inline lib/beaker/dsl/install_utils.rb]: These can be accomplished with host.add_env_var.

Comment (petems) [inline lib/beaker/dsl/install_utils.rb]: Fixed

**Comment by anodelman** [comment]
Needs a rebase.

**Comment by petems** [comment]
@anodelman Rebased :+1:

File: lib/beaker/dsl/install_utils/puppet_utils.rb [hunk 2]
```diff
@@ -369,15 +369,20 @@ def install_puppet_from_msi( host, opts )
           else
             dest = "C:\\Windows\\Temp\\#{host['dist']}.msi"
 
-            on host, "set PATH=\"%PATH%;#{host['puppetbindir']}\""
-            on host, "setx PATH \"%PATH%;#{host['puppetbindir']}\""
+            set_path_string = host['puppetbindir'].gsub(/"/,'')
```
Comment (anodelman) [inline lib/beaker/dsl/install_utils/puppet_utils.rb]: The puppetbindir is added to the env during configuration (in host_prebuilt_steps::set_env) so I think that this secondary setting is unnecessary.

**Comment by anodelman** [comment]
Review comment still current.

File: lib/beaker/dsl/install_utils/puppet_utils.rb (hunk 2 above)
Comment (petems) [inline lib/beaker/dsl/install_utils/puppet_utils.rb]: Fixed :+1:

**Pull request merged**